stylix/palette.nix

```
@@ -124,6 +124,13 @@ in {
   config = {
     # This attrset can be used like a function too, see
     # https://github.com/SenchoPens/base16.nix/blob/b390e87cd404e65ab4d786666351f1292e89162a/README.md#theme-step-22
+
+    assertions = lib.mkIf (cfg.image == null)
+      [ { assertion = cfg.image != null;
+          message = "One of `stylix.image` or `stylix.base16Scheme` must be set";
+        }
+      ];
```

This `cfg.image == null && cfg.image != null` assertion is weird. I think it should be something like:

```suggestion
    assertions = [
      {
        assertion = cfg.image != null || cfg.base16Scheme != null;
        message = "One of `stylix.image` or `stylix.base16Scheme` must be set";
      }
    ];
```

---

that looks a lot better. thank you